models/model.py

```diff
@@ -30,7 +30,9 @@ def __init__(self, model_hparams, optimizer_hparams):
 
         roberta_output_dim = 768
         model_name = model_hparams['model']
-        if model_name == 'roberta':
+        if 'checkpoint' in model_hparams and model_hparams['checkpoint'] is not None:
+            self.model = load_pretrained_encoder(model_hparams['checkpoint'])
```

Comment:
Why do we need to resume the training in a normal training setting (without transfer)? There is a difference between resuming and transferring because for transfer we freeze the weights of the encoder

Reply:
I think it was more like a "we need something to reload pretrained models". But then we need the preloaded models for two cases: 1. transfer and 2. Roberta GNN (which is not implemented yet). So maybe we could have 1 parser argument for the checkpoint and 1 parser argument for the "mode" for which we load it (either transfer or default the latter would simply load the checkpoint and do no freezing).

Reply:
Good idea, I will make a ```transfer``` argument which will be a flag to set when transferring.